suite-native/intl/src/en.ts

```diff
@@ -1005,6 +1005,23 @@ export const en = {
                 standard: 'Standard wallet',
                 defaultPassphrase: 'Passphrase wallet #{index}',
             },
+            autoEject: {
+                switch: {
+                    title: 'Auto eject',
+                    description: 'Eject all wallets automatically after unplugging Trezor',
```

Comment:
incorrect [description](https://www.figma.com/design/aybSBSnUWPebHVCpFvVtrj/Auto-eject?node-id=2204-6120&t=5OZyuuXRg5fxen1Z-1), should include also the number of wallets + `auto-eject`

Reply:
I guess you meant to tag this? 
https://github.com/trezor/trezor-suite/pull/20250/commits/1d6fbba9c7d376caed210c97613f53dfa51893c0